swarm/benches/connection_handler.rs

```diff
@@ -0,0 +1,359 @@
+use async_std::stream::StreamExt;
```

Comment:
do we need to add this file to the repository?

Reply:
Well it contains the benchmark @thomaseizinger asked me to write

Reply:
Where do you think it should live instead @jxs?

Reply:
I just want to mention that when this PR started, I was still heavily involved in `rust-libp2p` maintenance but this has since changed and I've now taken on a different role and @jxs is the primary maintainer now.

I think it is useful to have the benchmark here because it allows us to detect regressions when the `Connection` code is touched next. I'll leave it up to @jxs to decide whether it should be included in the end. Even if not, I still think it served its purpose because it has given us realistic numbers for what performance improvements users can expect.

Reply:
thanks for expanding Thomas! Makes sense